### ISSUE TITLE
the mathjax rendering norms and rendering problem

### ISSUE THREAD

**Issue by sylph520** [tamlok/vnote#240]
- VNote 1.16 with Windows 10 the latest
- Begin using vnote today, it's the vim mode within to try it, and other features are also quite nice. Great job!
- coz I need to edit many math expression, and when I come to Mathjax  support problems, I don't know how to write the math expression below 
![image](https://user-images.githubusercontent.com/4487630/39981879-1884ef66-5785-11e8-8f33-37c581f0969f.png)
well, even when I edit it well in another markdown editor, then open it with VNote, it seems unable to render correctly? 
- Have you guys encounter this situation? Thanks in advance.

**Comment by tamlok** [comment]
Hi,

Please try to wrap your formula definitions in a fenced code block with language `mathjax`:

    ```mathjax
    $$
    Definition here
    $$
    ```

If you write the MathJax definitions outside a code block, you need to escape some special characters by `\`, such as `\*`, `\_`, `\{`. I am working on `markdown-it` to try to get rid of escaping.

Thanks!

**Comment by sylph520** [comment]
thanks, that solves many problems, but still one issue left, could you please help me find where's the error in my mathjax code, some error exists in the conditioning area. ps. hoping for the _markdown-it_ 
![image](https://user-images.githubusercontent.com/4487630/39987643-effaa5f0-5797-11e8-9928-9b3dc2775bb9.png)

``` mathjax
$$
f(s, q)=\cases {b_{first}, &if q = 0 \\ b_{min}, &else if s > t_{adopt} \\b_{min}+[(b_{max} - b_{min}) \times (1- s/t_{adopt}))], &else if s > t_{sort} \\ b_{sort}, &else }
$$
```

**Comment by tamlok** [comment]
Hi,

Putting it in a fenced code block should work without escaping. If you do not want a code block, you need to escape the `_` and `\`:

```
$$
f(s, q)=\\cases {b\_{first}, &if q = 0 \\\\ b\_{min}, &else if s > t\_{adopt} \\\\b\_{min}+[(b\_{max} - b\_{min}) \\times (1- s/t\_{adopt}))], &else if s > t\_{sort} \\\\ b\_{sort}, &else }
$$
```

**Comment by tamlok** [comment]
Fixed in 1.20.
